jupyterhub/tests/test_named_servers.py

```diff
@@ -43,6 +43,7 @@ def test_default_server(app, named_servers):
                 'url': user.url,
                 'pending': None,
                 'progress_url': 'PREFIX/hub/api/users/{}/server/progress'.format(username),
+                'state': {},
```

Comment:
In this file some of these have the pid in the tests.  Needs to be ignored or matched somehow - in the right places.